examples/tests/valid/platforms/tf-backend.js

```diff
@@ -29,21 +29,21 @@
  * STATE_FILE - The object key used to store state file (default: wing-tf.tfstate)
  */
 
-// compatibleTargets not currently used see: https://github.com/winglang/wing/issues/1474
-exports.compatibleTargets = ["tf-aws", "tf-azure", "tf-gcp"]
-
-exports.postSynth = function(config) {
-  if (!process.env.TF_BACKEND) {throw new Error("env var TF_BACKEND not set")}
-
-  switch (process.env.TF_BACKEND) {
-    case "s3":
-      return configureS3Backend(config);
-    case "azurerm":
-      return configureAzurermBackend(config);
-    case "gcs":
-      return configureGcsBackend(config);
-    default:
-      throw new Error(`Invalid value for TF_BACKEND: ${process.env.TF_BACKEND} (valid values: s3, azurerm, gcs)`)
+exports.Platform = class TFBackend {
+  target = "tf-*";
```

Comment:
Nice! Is this tested?

Reply:
The TFBackend or the `tf-*`?

In the case of the the TFBackend, yes this has a test. 

As for the compatibility bit, no I have an open issue for enforcing compatibility: https://github.com/winglang/wing/issues/1474